sections/errorhandling/failfast.md

```diff
@@ -29,11 +28,11 @@ function addNewMember(newMember)
 ### Anti-pattern: no validation yields nasty bugs
 
 ```javascript
-// if the discount is positive let's then redirect the user to pring his discount coupons
-function redirectToPrintDiscount(httpResponse, member, discount)
-{
-    if(discount != 0)
+// if the discount is positive let's then redirect the user to print his discount coupons
+function redirectToPrintDiscount(httpResponse, member, discount) 

```

Comment:
Is the opening bracket of this function missing or is the diff view misleading?

Reply:
It's actually missing, just looked up the file in its current state